io.github.yesser_studios.Pong.yml

```diff
@@ -0,0 +1,51 @@
+id: io.github.yesser_studios.Pong
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+finish-args:
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+
+command: run.sh
+modules:
+  - name: dotnet
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/dotnet8/bin/install.sh
+
+  - name: pong
+    buildsystem: simple
+    build-options:
+      arch:
+        aarch64:
+          env:
+            RUNTIME: linux-arm64
+        x86_64:
+          env:
+            RUNTIME: linux-x64
```

Comment:
is this a leftover? I don't see where this is being used ?

Reply:
```suggestion
```

Reply:
> is this a leftover? I don't see where this is being used ?

Yes, this is a leftover.